docs/leaderboard/index.md

```diff
@@ -9,11 +9,11 @@ pagination_next: null
 
 ## What is Momento Leaderboard?
 
-Momento Leaderboard is a purpose-built leaderboard that supports tens of millions of items and rapid ingestion/querying/updates. While many databases approximate leaderboards via general purpose data structures like sorted set or range queries, Momento Leaderboard is a first-class, full-fledged service that you can quickly and easily integrate with your application. 
+Momento Leaderboardd is a purpose-built leaderboard service that supports tens of millions of items and rapid ingestion/querying/updates. While many databases approximate leaderboards via general purpose data structures like sorted set or range queries, Momento Leaderboards is a first-class, full-fledged service that you can quickly and easily integrate with your application.
```

Comment:
Come on, Pete! "Leaderboardd"?

Reply:
Dang. Was meant to be an "s" - I swear. Thanks!